### PR TITLE
change in width and align

### PR DESCRIPTION
**Pull request by yojana-tripathi**
i just change the width and align of the question no. and options and still change are pending in code of model so...check it

### CODE REVIEW

File: src/component/questions/QuestionSeries.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      correctAnswerIndex: 1,
+    },
+    {
+      question: "What year was JavaScript launched?",
+      options: ["1996", "1995", "1994", "none of the above"],
+      correctAnswerIndex: 1,
+    },
+    {
+      question: "What does CSS stand for?",
+      options: [
```
Comment (thakursachin05) [inline src/component/questions/QuestionSeries.jsx:24]: where is images in questions and where is images in options.

File: src/component/questions/QuestionSeries.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        "Cascading Simple Sheets",
+        "Cars SUVs Sailboats",
+      ],
+      correctAnswerIndex: 1,
+    },
+    {
+      question: "Which language runs in a web browser?",
+      options: ["Java", "C", "Python", "JavaScript"],
+      correctAnswerIndex: 3,
+    },
```
Comment (thakursachin05) [inline src/component/questions/QuestionSeries.jsx:36]: option object is not correct please check with the model used in the backend

File: src/component/questions/Question.jsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+                  id="collapseOne"
+                  class="accordion-collapse collapse show"
+                  aria-labelledby="headingOne"
+                  data-bs-parent="#accordionExample"
+                >
+                  <div class="accordion-body">
+                    <strong>This is the first item's accordion body.</strong>
+                  </div>
+                </div>
+              </div>
```
Comment (thakursachin05) [inline src/component/questions/Question.jsx:88]: we cannot show hardcoded explaination it should be coming from the api data , please refer to the backend model

**Review by thakursachin05 — CHANGES_REQUESTED** [review]
Please make the changes

File: src/component/questions/Question.jsx (hunk 3 above)
Comment (yojana-tripathi) [reply src/component/questions/Question.jsx:88]: okayy i will

File: src/component/questions/QuestionSeries.jsx (hunk 2 above)
Comment (yojana-tripathi) [reply src/component/questions/QuestionSeries.jsx:36]: this is previous code i use to deit the width and align only this is not the correct code of model

Comment (yojana-tripathi) [reply src/component/questions/QuestionSeries.jsx:36]: edit*

**Pull request closed without merging**